### PR TITLE
fix(llm): 🐛 fix long Memo Tag issue on Stacks

### PR DESCRIPTION
**Pull request by thesan**
### ✅ Checklist

- [x] `npx changeset` was attached.
- [ ] **Covered by automatic tests.** 
- [ ] **Impact of the changes:** 
  - ...

### 📝 Description

Truncate Stacks memos in the input to prevent the transaction validation from failing. Follow up to #8340.

https://github.com/user-attachments/assets/b917558b-8872-44c1-962b-478e36bb20e4

### ❓ Context

- [LIVE-15164]

---

### 🧐 Checklist for the PR Reviewers

- **The code aligns with the requirements** described in the linked JIRA or GitHub issue.
- **The PR description clearly documents the changes** made and explains any technical trade-offs or design decisions.
- **There are no undocumented trade-offs**, technical debt, or maintainability issues.
- **The PR has been tested** thoroughly, and any potential edge cases have been considered and handled.
- **Any new dependencies** have been justified and documented.
- **Performance** considerations have been taken into account. (changes have been profiled or benchmarked if necessary)

[LIVE-15164]: https://ledgerhq.atlassian.net/browse/LIVE-15164?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/ledger-live-mobile/src/families/stacks/MemoTagInput.tsx [hunk 1]
```diff
@@ -1,12 +1,14 @@
...
 
+import { STACKS_MAX_MEMO_SIZE } from "@ledgerhq/live-common/families/stacks/constants";
 import type { Transaction as StacksTransaction } from "@ledgerhq/live-common/families/stacks/types";
 import type { MemoTagInputProps } from "LLM/features/MemoTag/types";
 import { GenericMemoTagInput } from "LLM/features/MemoTag/components/GenericMemoTagInput";
 
 export default (props: MemoTagInputProps<StacksTransaction>) => (
   <GenericMemoTagInput
     {...props}
+    textToValue={text => text.slice(0, STACKS_MAX_MEMO_SIZE)}
```
Comment (KVNLS) [inline apps/ledger-live-mobile/src/families/stacks/MemoTagInput.tsx:11]: Not sure if we want to have this solution, can you confirm with our PM, please?

Comment (thesan) [reply apps/ledger-live-mobile/src/families/stacks/MemoTagInput.tsx:11]: Yes I double checked

**Review by ComradeAERGO — DISMISSED** [review]
LGTM

**Review by KVNLS — APPROVED** [review]

**Comment by thesan** [comment]
@KVNLS @ComradeAERGO thank you for the reviews but I will have to edit this PR because the size limit is not 34 chars is 34 bytes so the curent solution doesn't work for non ASCII characters:

https://github.com/user-attachments/assets/f97d44f4-d2bf-49b7-9e5e-2d454ac44e35

**Review by KVNLS — APPROVED** [review]

**Review by ComradeAERGO — APPROVED** [review]

**Pull request merged**